bug: EXDEV: cross-device link not permitted when moving the Kind binary

```
with:
    version: v0.7.0
    name: kind
    wait: 300s
    skipClusterCreation: false
  env:
    GOROOT: /opt/hostedtoolcache/go/1.13.11/x64
downloading kind from https://github.com/kubernetes-sigs/kind/releases/download/v0.7.0/kind-linux-amd64
chmod +x /home/runner/work/_temp/174cfcc2-60fa-4457-beaa-ccc15123e9e8
##[error]EXDEV: cross-device link not permitted, rename '/home/runner/work/_temp/174cfcc2-60fa-4457-beaa-ccc15123e9e8' -> '/home/runner/bin/kind'
```

https://github.com/crossplane/addon-oam-kubernetes-local/runs/697514545?check_suite_focus=true

--- a/README.md
+++ b/README.md
@@ -13,7 +13,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@master
-    - uses: engineerd/setup-kind@v0.3.0
+    - uses: engineerd/setup-kind@v0.4.0
     - name: Testing
       run: |
         kubectl cluster-info
@@ -49,7 +49,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@master
-    - uses: engineerd/setup-kind@v0.3.0
+    - uses: engineerd/setup-kind@v0.4.0
       with:
           version: "v0.7.0"
     - name: Testing

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "setup-kind",
-  "version": "0.1.0",
+  "version": "0.4.0",
   "private": true,
   "description": "GitHub Action to setup KinD (Kubernetes in Docker)",
   "main": "lib/main.js",

--- a/package.json
+++ b/package.json
@@ -20,10 +20,9 @@
   "author": "Engineerd",
   "license": "MIT",
   "dependencies": {
-    "@actions/core": "^1.0.0",
-    "@actions/tool-cache": "^1.1.0",
-    "@actions/exec": "^1.0.0",
-    "@actions/io": "^1.0.0"
+    "@actions/core": "^1.2.4",
+    "@actions/tool-cache": "^1.5.5",
+    "@actions/exec": "^1.0.4"
   },
   "devDependencies": {
     "@types/chai": "^4.0.1",

--- a/test/action.ts
+++ b/test/action.ts
@@ -38,6 +38,4 @@ describe("checking input parsing", function () {
         let args: string[] = getKindConfig().getCommand();
         assert.deepEqual(args, ["create", "cluster", "--config", "/home/runner/repo/some-path", "--image", testEnvVars.INPUT_IMAGE, "--name", testEnvVars.INPUT_NAME, "--wait", testEnvVars.INPUT_WAIT]);
     });
-
-
 });

